### DISCUSSION TITLE
Is Shizuku safe and will it mess up the phone?

### DISCUSSION THREAD

**Discussion by mduong1981** [RikkaApps/Shizuku#299] [General]
I ask this question because on my Samsung Galaxy S20+ one day I woke up and the alarm was working fine and the screen turned on so that I could hit the snooze button. Then when I finally woke up 15 to 20 minutes afterwards my phone screen blacked out and it still has sound. Could Shizuku be the cause of this problem? Someone please help and let me know because it was the first time I ever used it. Now I'm thinking about it again on this Motorola Moto g60 because the customizations were really fun and cool. I did freeze some apps with Ice Box when I used it last. So just trying to see if anyone else had this problem before? Thank you community ❤️!

**Comment by thejaustin** [comment]
> I ask this question because on my Samsung Galaxy S20+ one day I woke up and the alarm was working fine and the screen turned on so that I could hit the snooze button. Then when I finally woke up 15 to 20 minutes afterwards my phone screen blacked out and it still has sound. Could Shizuku be the cause of this problem? Someone please help and let me know because it was the first time I ever used it. Now I'm thinking about it again on this Motorola Moto g60 because the customizations were really fun and cool. I did freeze some apps with Ice Box when I used it last. So just trying to see if anyone else had this problem before? Thank you community ❤️!

Most likely has something to do with the apps you disabled with Ice Box. I'd review those and ensure none were system apps or anything. Rebooting your phone in safe mode or just rebooting in general may get you back into your device. I recommend Hail for app freezing myself since it still receives consistent updates!

**Reply by mduong1981** [reply]
Thank you so much 😀! Yes I did freeze some system apps. But it did work well for a couple of weeks before that happened. Also the Samsung Galaxy S20+ did get dropped in water for like a second and pulled back up. So I'm not sure what it could be or some system apps being disabled. Also I have read that Samsung Galaxy S20s seem to have a black screen come on sometimes and then needs repair 🤷. But thank you for the input. I really greatly appreciate your timely response and help because I really love tinkering around with my devices. Have a wonderful evening.
